### PR TITLE
874597 - Added Maui RadialMenu control UG content

### PR DESCRIPTION
**Pull request by prakashKannanSf3972**
## Description ##

- Added Radial menu control UG content.

### CODE REVIEW

File: MAUI/Radial-Menu/Overview.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The Syncfusion .NET MAUI SfRadialMenu displays a hierarchical menu in a circular layout, which is optimized for touch enabled devices. Typically, it is used as a context menu, and it can expose more menu items in the same space than traditional menus.
+
+## Key features
+
+`Drag` : The SfRadialMenu can be floated over the layout to avoid obscuring the content behind it.
+`Rotation` : SfRadialMenu supports rotating items.
+`FontIcon`: The built-in icon font option helps users add vector images that prevent the control from experiencing any image glitches often faced with traditional image icons.
+`Custom view` : SfRadialMenu supports the custom view like image.
+`Custom segments` : Complete customization options for the menu and its items, such as coloring, sizing,palcement and shapes, using the segmentation option.
```
Comment (M-A32) [inline MAUI/Radial-Menu/Overview.md:20]: `Custom segments` : Complete customization options for the menu and its items, such as coloring, sizing, placement, and shapes, using the segmentation option.

File: MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+## DragEvents
+
+[`SfRadialMenu`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html) provides for event for [`DragBegin`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html#Syncfusion_Maui_SfRadialMenu_DragBegin) and [`DragEnd`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html#Syncfusion_Maui_SfRadialMenu_DragEnd) in which the event get hooked when the RadialMenu is get dragged.
+
+### DragBegin event
+
+This event get triggered when RadialMenu is start to drag with [`DragBeginEventArgs`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html).
+
+* [`Position`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html#Syncfusion_Maui__SfRadialMenu_DragBeginEventArgs_Position) - Gets the Start position of the RadialMenu
```
Comment (M-A32) [inline MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md:82]: * [`Position`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html#Syncfusion_Maui__SfRadialMenu_DragBeginEventArgs_Position) - Gets the Start position of the RadialMenu.

File: MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Drag End
+
+This event get triggered when dragging is ended in RadialMenu with [`DragEndEventArgs`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html).
+
+* [`OldValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html#Syncfusion_Maui_SfRadialMenu_DragEndEventArgs_OldValue) - Gets the Start position of the RadialMenu
```
Comment (M-A32) [inline MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md:150]: * [`OldValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html#Syncfusion_Maui_SfRadialMenu_DragEndEventArgs_OldValue) - Gets the Start position of the RadialMenu.

File: MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+* [`NewValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html#Syncfusion_Maui_SfRadialMenu_DragEndEventArgs_NewValue) - Gets the ens position of the RadialMenu
```
Comment (M-A32) [inline MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md:152]: * [`NewValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html#Syncfusion_Maui_SfRadialMenu_DragEndEventArgs_NewValue) - Gets the end position of the RadialMenu.

File: MAUI/Radial-Menu/Populating-Items.md [hunk 5]
```diff
@@ -0,0 +1,723 @@
+---
+layout: post
+title: Populating Items in MAUI Radial Menu control | Syncfusion
```
Comment (M-A32) [inline MAUI/Radial-Menu/Populating-Items.md:3]: title: Populating Items in MAUI Radial Menu Control | Syncfusion

File: MAUI/Radial-Menu/Populating-Items.md [hunk 6]
```diff
@@ -0,0 +1,723 @@
...
+
+This section explains the ways about populating items through radial menu item and item source with item template.
+
+## Through radial menu items
+
+By passing a collection of [`SfRadialMenuItem`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadialMenuItem.html), you can get the view of [`SfRadialMenu`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadioButton.html) control. The radial menu item class provides various options to customize the items by giving custom views, font icons, and images. You can add radial menu items by hierarchy.
+
+### Text
+
+The following code snippet demonstrates how to Create a RadialMenu control with Items having the given text.
```
Comment (M-A32) [inline MAUI/Radial-Menu/Populating-Items.md:20]: The following code sample demonstrates how to Create a RadialMenu control with Items having the given text.

File: MAUI/Radial-Menu/Populating-Items.md [hunk 7]
```diff
@@ -0,0 +1,723 @@
...
+
+{% endtabs %}
+
+N> Instead of using ObservableCollection, use `RadialMenuItemsCollection` for the RadialMenu Items list and `SubMenuItemsCollection` for the list of items within each RadialMenuItem.
+
+![Display text in .NET MAUI RadialMenu](images/populating-items/maui-radialmenu-populating-only-text.png)
+
+### ImageWithText
+
+The following code snippet demonstrates how to Display images and text in the RadialMenu items of the control.
```
Comment (M-A32) [inline MAUI/Radial-Menu/Populating-Items.md:83]: The following code sample demonstrates how to Display images and text in the RadialMenu items of the control.

File: MAUI/Radial-Menu/Populating-Items.md [hunk 8]
```diff
@@ -0,0 +1,723 @@
...
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Custom Font with Text
+
+The following code snippet demonstrates how to Display custom font with text in the RadialMenu items of the control.
```
Comment (M-A32) [inline MAUI/Radial-Menu/Populating-Items.md:144]: The following code sample demonstrates how to Display custom font with text in the RadialMenu items of the control.

File: MAUI/Radial-Menu/Populating-Items.md [hunk 9]
```diff
@@ -0,0 +1,723 @@
...
+
+{% endtabs %}
+
+![Display font with text in .NET MAUI RadialMenu.](images/populating-items/maui-radialmenu-populating-text-icons.png)
+
+### Adding nested items of radial menu
+
+You can populate the nested levels of items within a menu to group similar actions based on their result. For example, you can group the clipboard operations by adding a clipboard as a main menu and cut, copy, and paste as its children.
+
+The following code snippet demonstrates how to add the nested items of radial menu.
```
Comment (M-A32) [inline MAUI/Radial-Menu/Populating-Items.md:223]: The following code sample demonstrates how to add the nested items of radial menu.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**